Source/BusinessMomentsAdapters/Web/configuration/BasicAuthComponent.tsx

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+import React from 'react';
+import { TextField, ITextFieldStyles } from '@fluentui/react/lib/TextField';
+import { Label } from '@fluentui/react/lib/Label';
+import { Stack } from '@fluentui/react/lib/Stack';
+
+const textFieldStyles: Partial<ITextFieldStyles> = { fieldGroup: { width: 300 } };
+const stackTokens = { childrenGap: 15 };
+
+export const BasicAuthComponent: React.FunctionComponent = () => {
```

Comment:
You don't need to include the type `React.FunctionComponent`.

```suggestion
export const BasicAuthComponent = () => {
```